### PR TITLE
Add polish

### PR DESCRIPTION
**Pull request by MAX0342**
That's it

### CODE REVIEW

File: texts/index.js [hunk 1]
```diff
@@ -265,3 +265,41 @@ texts.push([
...
+	"Vita3K to pierwszy na świecie experymentalny i otwartoźródłowy emulator PlayStation Vity dla Windows-a i macOS-a.", // 10
+	"Pamietaj, że celem tego emulatora nie jest umożliwienie nielegalnej aktywnośći.", // 11
+	"Wybierz swoją kompilacje!", // 12
+	"Dostępne wkrótce!", // 13
+	"Weż z", // 14
+	"Skontaktuj się z nami!", // 15
+	"Skontaktuj się z deweloperami i innymi urzytkownikami kiedy tylko chcesz. Wolimy przez Discord, ale to twój wybór!", // 16
+	"Dołącz!", // 17
+	"Sprawdź co działa!", // 18
+	"Ten emulator jeszcze nie obsługuje <b>żadnych gier komercyjnych</b>.Wiele gier homebrew są obsługiwane. Proszę sprawdzić <a href=\"compatibility.html?lang=en\">liste kompatybilnośći</a> po więcej detali!", // 19
```
Comment (VelocityRa) [inline texts/index.js]: Change `lang=en` to `lang=pl`.

Comment (MAX0342) [reply texts/index.js]: ok

Comment (MAX0342) [reply texts/index.js]: where

Comment (MAX0342) [reply texts/index.js]: oh here

**Pull request merged**